src/pages/HomePage.tsx

```
@@ -288,98 +387,185 @@ export const HomePage: React.FC = () => {
         </div>
       </section>
 
-      <section id="galleries" ref={galleriesRef as React.RefObject<HTMLElement>}>
+      <section id="portfolio-hub" ref={galleriesRef as React.RefObject<HTMLElement>}>
         <div className="container">
-          <h2 className="section-heading">Showcases</h2>
-          <div className="galleries-grid">
-            <div className="gallery-card">
-              <div className="gallery-icon">
-                <i className="fas fa-graduation-cap"></i>
+          <h2 className="purpose-heading">// Why I Built This Portfolio</h2>
```

Biome (2.1.2)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)